fe/fe-core/src/main/java/com/starrocks/common/util/RuntimeProfile.java

```diff
@@ -343,7 +343,14 @@ public String toString() {
         return formater.format(this);
     }
 
-    private String printCounter(long value, TUnit type) {
+    public static String printCounter(Counter counter) {
```

Comment:
What's the purpose of `explain analyze`? It's better to add a commit message

Reply:
This pr is under developmenet now, I've changed the title, and I'll add description after finishing this pr.